Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/Render/MarkdownRenderer.Blocks.cs

```diff
@@ -0,0 +1,432 @@
+            }
+
+            var blockUIElementCollection = localContext.BlockUIElementCollection;
+
+            var brush = localContext.Foreground;
+            if (HorizontalRuleBrush != null && !localContext.OverrideForeground)
+            {
+                brush = HorizontalRuleBrush;
+            }
+
+            var rectangle = new Rectangle
+            {
+                HorizontalAlignment = HorizontalAlignment.Stretch,
+                Height = HorizontalRuleThickness,
+                Fill = brush,
+                Margin = HorizontalRuleMargin
+            };
+
+            blockUIElementCollection.Add(rectangle);
+        }
+
+        /// <summary>
+        /// Renders a quote element.
+        /// </summary>
+        protected override void RenderQuote(QuoteBlock element, IRenderContext context)
+        {
+            var localContext = context as UIElementCollectionRenderContext;
+            if (localContext == null)
+            {
+                throw new RenderContextIncorrectException();
+            }
+
+            var blockUIElementCollection = localContext.BlockUIElementCollection;
+
+            var stackPanel = new StackPanel();
+            var childContext = new UIElementCollectionRenderContext(stackPanel.Children, context)
+            {
+                Parent = stackPanel
+            };
+
+            if (QuoteForeground != null && !localContext.OverrideForeground)
+            {
+                childContext.Foreground = QuoteForeground;
+            }
+
+            RenderBlocks(element.Blocks, childContext);
+
+            var border = new Border
+            {
+                Margin = QuoteMargin,
+                Background = QuoteBackground,
+                BorderBrush = childContext.OverrideForeground ? childContext.Foreground : QuoteBorderBrush ?? childContext.Foreground,
+                BorderThickness = QuoteBorderThickness,
+                Padding = QuotePadding,
+                Child = stackPanel
+            };
+
+            blockUIElementCollection.Add(border);
+        }
+
+        /// <summary>
+        /// Renders a code element.
+        /// </summary>
+        protected override void RenderCode(CodeBlock element, IRenderContext context)
+        {
+            var localContext = context as UIElementCollectionRenderContext;
+            if (localContext == null)
+            {
+                throw new RenderContextIncorrectException();
+            }
+
+            var blockUIElementCollection = localContext.BlockUIElementCollection;
+
+            var brush = localContext.Foreground;
+            if (CodeForeground != null && !localContext.OverrideForeground)
+            {
+                brush = CodeForeground;
+            }
+
+            var textBlock = new RichTextBlock
+            {
+                FontFamily = CodeFontFamily ?? FontFamily,
+                Foreground = brush,
+                LineHeight = FontSize * 1.4
```

Comment:
Magic Number?

Reply:
As discussed offline, was here before, we can worry about another day.

Reply:
This was an existing implementation.